src/Configuration.php

```diff
@@ -134,7 +137,14 @@ public static function load(string $path = null, array $paths = []): self
         $filesFromPaths = self::retrieveFilesFromPaths($paths);
         $filesWithContents = self::retrieveFilesWithContents(chain($filesFromPaths, ...$finders));
 
-        return new self($path, $prefix, $filesWithContents, $patchers, $whitelist, $whitelistedFiles);
+        if (false === array_key_exists(self::OUTPUT_DIR_KEYWORD, $config)) {
+            $outputDir = null;
+        }
+        else {
```

Comment:
I would keep it on the same line:

```suggestion
       } else {
```

Same for the other occurrence